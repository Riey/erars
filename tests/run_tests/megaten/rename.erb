@SYSTEM_TITLE
DE:3:0 = 123
PRINTFORML 권유할 자를 고르세요　　　경배치：{[[점포:인기]]:ARG}
