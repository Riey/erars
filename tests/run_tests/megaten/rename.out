권유할 자를 고르세요　　　경배치：123
